Sources/xcproj/PBXProjObjects+Helpers.swift

```diff
@@ -15,7 +16,160 @@ public extension PBXProj.Objects {
         targets.append(contentsOf: Array(aggregateTargets.values) as [PBXTarget])
         return targets.filter { $0.name == name }
     }
-    
+
+    /// Retruns target's sources build phase.
+    ///
+    /// - Parameter target: target object.
+    /// - Returns: target's sources build phase, if found.
+    public func sourcesBuildPhase(target: PBXTarget) -> PBXSourcesBuildPhase? {
```

Comment:
Nice additions @ilyapuchka 👏

Reply:
`sourcesBuildPhases` is deprecated. Instead you should use `objects.sourcesBuildPhases`.

Reply:
this is exactly what is being used, this is extension on PBXObjects.

Reply:
👍 sorry, missed that.